docs/Internal Systems/Research & Findings/StaticSiteGeneratorsAreeb.md

```diff
@@ -0,0 +1,77 @@
+# Static Site Generator 
+
+
+_The purpose of this document is to decide if Gatsby is the most suitable static site generator for our project._
+
+ <br>
+
+Static site are used in cases where you only need to spit out content without the use of back-end frameworks or databases. Use cases could be blog posts, documentations or landing pages. In our case, we are building something similar to a documentation page where we are collecting all the information of Thoth Tech in one single place. I have narrowed down three site generators that could be used for our project 
+
+ 
+ <br>
+
+ 
+
+# 
+ <br>
+
+## Gatsby  
+<br>
+
+* Gatsby is a site generator based on JavaScript and React 
```

Comment:
Please use `-` as a bullent for consistency across all documentation

Reply:
Done